[DOCU-2188] Add docs for Prometheus 1.5.0 (#3666)
Kong/docs.konghq.com: app/_hub/kong-inc/prometheus/index.md

data plane nodes is exported to control plane.
- **Enterprise License Information**: The {{site.base_gateway}} license expiration date, features and
license signature. Those metrics are only exported on {{site.base_gateway}}.

> - **DB Entity Count**: A gauge metric that measures the current number of
>     database entities (**Enterprise-Only**).

Reviewer: ```suggestion
- **DB Entity Count** <span class="badge enterprise"></span> : A gauge metric that 
    measures the current number of database entities.
```

> 

Reviewer: Should there also be an error count metric (kong_db_entity_count_errors)?

Author: I left it out because it's kind of a meta-metric, and this section seems like it's more of a general feature overview rather than an exhaustive reference. Though, I have no qualms about putting it in here if you think that's the way to go.

Reviewer: You're right, none of these metric descriptions actually have the specific metric names, so that makes sense. Both should be covered with the sample output anyway.

> Here is an example of output you could expect from the `/metrics` endpoint:

Reviewer: If possible, can you add sample output for the two new metrics to the codeblock below?

* New `data_plane_cluster_cert_expiry_timestamp` metric
* Added `subsystem` label to Upstream Target health metrics

### 1.5.x

* Add two new metrics:

>   * `kong_db_entities_total` (gauge) total number of entities in the database

Reviewer: ```suggestion
  * `kong_db_entities_total` (gauge): total number of entities in the database
```

>   * `kong_db_entity_count_errors` (counter) measures the number of errors

Reviewer: ```suggestion
  * `kong_db_entity_count_errors` (counter): measures the number of errors
```
